extensions.html

```diff
@@ -137,7 +137,17 @@ <h2>Introduction</h2>
 
     </section>
 
-    <section id='conformance'></section>
+    <section id='conformance'>
+
+      <h2>Security and privacy considerations</h2>
+
+      <p>User agents MUST implement the <a>GamepadTouch</a> to only be
+      available in <a data-cite= "secure-contexts#secure-context">secure
+      contexts</a>. This provides better protection for the user against
+      man-in-the-middle attacks intended to obtain gamepad data. Browsers may
+      ignore this rule for development purposes only.</p>
+
+    </section>
```

Comment:
Don't need this text. Just having the [SecureContext] extended attribute already implies all this and more. 
 
```suggestion
    <section id='conformance'></section>
```